config/main.py

```diff
@@ -4413,6 +4549,107 @@ def delete(ctx):
     sflow_tbl['global'].pop('agent_id')
     config_db.set_entry('SFLOW', 'global', sflow_tbl['global'])
 
+#
+# set ipv6 link local mode on a given interface
+#
+def set_ipv6_link_local_only_on_interface(config_db, interface_dict, interface_type, interface_name, mode):
+
+    curr_mode = config_db.get_entry(interface_type, interface_name).get('ipv6_use_link_local_only')
+    if curr_mode is not None:
+        if curr_mode == mode:
+            return
+    else:
+        if mode == "disable":
+            return
+
+    if mode == "enable":
+        config_db.mod_entry(interface_type, interface_name, {"ipv6_use_link_local_only": mode})
+        return
+
+    # If we are disabling the ipv6 link local on an interface, and if no other interface
+    # attributes/ip addresses are configured on the interface, delete the interface from the interface table
+    exists = False
+    for key in interface_dict.keys():
+        if not isinstance(key, tuple):
+            if interface_name == key:
+                #Interface bound to non-default-vrf do not delete the entry
+                if 'vrf_name' in interface_dict[key]:
+                    if len(interface_dict[key]['vrf_name']) > 0:
+                        exists = True
+                        break
+            continue
+        if interface_name in key:
+            exists = True
+            break
+
+    if exists:
+        config_db.mod_entry(interface_type, interface_name, {"ipv6_use_link_local_only": mode})
+    else:
+        config_db.set_entry(interface_type, interface_name, None)
+
+#
+# 'ipv6' group ('config ipv6 ...')
+#
+
+@config.group()
+@click.pass_context
+def ipv6(ctx):
+    """IPv6 configuration"""
+
+#
+# 'enable' command ('config ipv6 enable ...')
+#
+@ipv6.command()
+@click.pass_context
+def enable(ctx):
```

Comment:
i think this command should be enable like "`config ipv6 enable link-local`" instead of just enable

Reply:
Done, updated the command.